test/eventhub.test.ts

```diff
@@ -129,3 +135,11 @@ test("Test that unsubscribeAll unsubscribes all subscribed topics", () => {
   expect(eventhub.isSubscribed("testTopic3")).toEqual(false);
   expect(eventhub.isSubscribed("testTopic4")).toEqual(false);
 });
+
+test("Test that reconnect event is emitted", () => {
+  testServer.close();
+
+  setTimeout(()=>{
+    expect(emitEventSpy).toHaveBeenCalledWith('reconnect');
+  }, 100)
```

Comment:
correct me if I'm wrong, but  this test will be always green, cause jest does not wait for async callbacks to be completed. You need to either wrap it with Promise or use `done` callback to say explicitly when the test is done

Reply:
🙇 you are right, fixed!